src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -70,31 +70,26 @@ public ReplicatorTest()
             ReopenDB();
             Database.Delete("otherdb", Directory);
             _otherDB = OpenDB("otherdb");
-            Database.SetLogLevel(LogDomain.Replicator, LogLevel.Verbose);
+            //uncomment the code below when you need to see more detail log
+            //Database.SetLogLevel(LogDomain.Replicator, LogLevel.Verbose);
         }
 
         [Fact]
-        public async Task TestCloseServerWhenReplicatorExistsAsync()
+        public async Task TestToCheckReplicatorStopsWhenAttemptToConnectToInvalidEndpoint()
```

Comment:
This name is a bit long and I don't think it will show up in the test explorer without making it really wide.  How about `TestReplicatorStopsWhenEndpointInvalid`